.github/CODEOWNERS

```diff
@@ -2,3 +2,7 @@
 # .github/workflows directory (and the merge logic) using CODEOWNERS.
 /.github/workflows/ @klizhentas @russjones @r0mant @zmb3 @fheinecke @camscale @tcsc @rosstimothy
 /build.assets/tooling/cmd/difftest/ @klizhentas @russjones @r0mant @zmb3
+
+# Owners for dependency updates in JS packages.
+web/packages/teleterm/package.json @gzdunek @ravicious
+package.json @avatus @gzdunek @ravicious
```

Comment:
Do repo admins have total power? Do their approval count as code owner approvals, even when they're not specifically listed as code owners for a specific file?

I couldn't find info about it in the docs, but it seems reasonable. If that's not the case, then I think it might be a good idea to add another person from the other side of the globe.